src/gridList.test.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import GridList from './GridList';
+
+const JSX_GridListStatic = (
+	<GridList
+		columns={{
+			default: 3
+		}}
+		items={[
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>
+		]}
+	/>
+);
+
+const responsiveColsDefault = 2,
+	responsiveColsMedium = 4,
+	responsiveColsLarge = 6;
+
+const JSX_GridListResponsive = (
+	<GridList
+		columns={{
+			default: responsiveColsDefault,
+			medium: responsiveColsMedium,
+			large: responsiveColsLarge
+		}}
+		items={[
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>
+		]}
+	/>
+);
+
+let gridList,
+	gridListNode;
+
+describe('Static GridList', function() {
+	beforeEach(() => {
+		gridList = TestUtils.renderIntoDocument(JSX_GridListStatic);
+		gridListNode = ReactDOM.findDOMNode(gridList);
+	});
+	afterEach(() => {
+		gridList = null;
+		gridListNode = null;
+	});
+
+	it('exists', function() {
+		expect(gridListNode).not.toBeNull();
+	});
+
+	it('wraps gridList items with element containing className "gridList-item"', function() {
+		const glItems = TestUtils.scryRenderedDOMComponentsWithClass(gridList, 'gridList-item');
+		expect(glItems.length).not.toBe(0);
+	});
+});
+
+describe('Responsive GridList', function() {
+	let glClassList;
+
+	beforeEach(() => {
+		gridList = TestUtils.renderIntoDocument(JSX_GridListResponsive);
+		gridListNode = ReactDOM.findDOMNode(gridList);
```

Comment:
@eilinora shared a nice article in another PR related to this: http://reactkungfu.com/2015/07/approaches-to-testing-react-components-an-overview/

One of the principles is that you should 'maintain the component abstraction in your tests as long as possible, but no longer', i.e. you should definitely _prefer_ `TestUtils` methods that scan the React tree, but if you really need to get details of the DOM, you can drop into the 'real' DOM for some tests. I think one of the challenges of working with React is breaking the mental connection between 'React component tree' and 'DOM tree', and then determining at each step of application development _which_ of those abstractions you are desiging/testing.

The article is good, though - definitely worth your time.